cmd/helm/install_test.go

```diff
@@ -196,7 +196,6 @@ func TestInstall(t *testing.T) {
 			name:      "install with schema file, extra values from yaml, with errors",
 			cmd:       "install schema testdata/testcharts/chart-with-schema -f testdata/testcharts/chart-with-schema/extra-values.yaml",
 			wantError: true,
-			golden:    "output/schema-negative.txt",
```

Comment:
This `golden` file is only used here. If it's not going to be used anymore, the schema-negative.txt file should be removed.

Why is the `golden` file being removed?